src/chart/animator/options.h

```diff
@@ -25,7 +25,8 @@ enum class SectionId {
 	y,
 	x,
 	tooltip,
-	connection
+	connection,
+	nothing
```

Comment:
If we add nothing here, then we will have a nothing option group in the anim options object, so the user can say this:
```
chart.animate({...}, { nothing: { delay: x, duration: y, easing: whatever })
```
I don't think that we need this on the API.

Reply:
OK, I'll find another way.